docs/deployment/frameworks/hf_inference_endpoints.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Hugging Face Inference Endpoints
+---
+
+# Deploying Hugging Face Models with vLLM and Inference Endpoints
+
+## Overview
+
+Models compatible with vLLM can be deployed on Hugging Face Inference Endpoints, either starting from the [Hugging Face Hub](https://huggingface.co) or directly from the [Inference Endpoints](https://endpoints.huggingface.co/) interface. This allows you to serve models in a fully managed environment with GPU acceleration, auto-scaling, and monitoring, without managing the infrastructure manually.
+
+Additionally, with the [transformers backend integration](https://blog.vllm.ai/2025/04/11/transformers-backend.html), vLLM now provides Day 0 support for any model with `transformers` compatibility.
+
+Hugging Face Inference Endpoints provides a managed environment for serving models via vLLM. You can deploy models without configuring servers, installing dependencies, or managing clusters. Endpoints also allows deployment across multiple cloud providers (AWS, Azure, GCP) without needing separate accounts for each. They integrate seamlessly with the Hub: you can deploy any model compatible with vLLM or `transformers`, track usage, and update the inference engine directly. The vLLM engine is available out-of-the-box, allowing you to leverage optimized inference and switch between different models or engines without modifying your code. This simplifies production deployment: endpoints are ready in minutes, provide monitoring and logging, and let you focus on serving models rather than maintaining infrastructure.
+
+## Deployment Methods
+
+- [**Method 1: Deploy from the Catalog.**](#method-1-deploy-from-the-catalog) One-click deploy models from the Hugging Face Hub with ready-made optimized configurations.
+- [**Method 2: One-Click Deployment (Transformers Models).**](#method-2-one-click-deployment-transformers-models) Instantly deploy models tagged with `transformers` from the Hub UI using the **Deploy** button.
```

Comment:
Here I'd avoid using the term "one-click" deploy since there might be some configuration to do for the user. The defaults are best guesses vs. from the catalog the defaults are guaranteed to work.

Reply:
Changed to `Method 2: Guided Deployment (Transformers Models)`